python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -147,6 +147,12 @@ def test_basic(self, serve_instance):
             },
             max_ongoing_requests=25,
             version="v1",
+            # With RAY_SERVE_COLLECT_AUTOSCALING_METRICS_ON_HANDLE=1, ongoing requests may be
+            # double-counted, causing the count to exceed 50. This can temporarily trigger
```

Comment:
same question as above, why are ongoing requests double counted?

Reply:
addressed in other thread